.github/workflows/generate.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         continue-on-error: true
         run: |
             git checkout -b $BRANCH_NAME
-            [[ $(git diff --name-only) == "src/connectors/mod.ts" ]] && { echo "No real changes. Do not commit."; exit 1; }
+            [[ $(git ls-files --other --modified --exclude-standard) == "src/connectors/mod.ts" ]] && { echo "No real changes. Do not commit."; exit 1; }
```

Comment:
This change comes from #21 we may want to wait until this PR is merges before merging this